### ISSUE TITLE
Intel T265 tracking camera disconnects from Rpi 4B after one minute of operation

### ISSUE THREAD

**Issue by TCIII** [autorope/donkeycar#838]
Hardware: Rpi 4B 4GB
OS: Buster Desktop
Intel RealSense2 SDK: 2.44
T265 firmware: 0.2.0.951
Donkey Car: 4.1.0

Issue:
When running the realsense-viewer in SDK 2.44 on the Buster Desktop to view the fisheye cameras and the imu streams, the T265 will no longer respond to frame requests after running for about a minute or so.
This disconnect issue also results in the failure of the robot car to follow a recorded path beyond one minute of operation when using a mycar configuration created using the DC "path_follow" template.

Solution:
This issue appears to be limited to the Rpi 4B and presently there is no known solution other than to use a Rpi 3B+ or a NVIDIA Nano, etc.
It should be noted that the T265 is at the end of life as Intel has moved on to the D400 and up family of cameras.
Bottom line: Don't expect Intel to correct this problem with the Rpi 4B, however we might get lucky as some enterprising coder might come up with a useable solution.

**Comment by Ezward** [comment]
Given that the T265 is no longer supported by Intel, I am going to close this.
